### PR TITLE
Bug 831953 - Wifi tethering use now unique network name.

### PR DESCRIPTION
**Pull request by HafidTraikzi**

### CODE REVIEW

File: apps/settings/js/panels/hotspot/hotspot_settings.js [hunk 1]
```diff
@@ -107,6 +108,26 @@ define(function(require) {
       return password;
     },
 
+    _generateHotspotSSID: function hs_generateHotspotSSID() {
+      var characters = 'abcdefghijklmnopqrstuvwxyz' +
+      'ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890';
+      var password = 'FirefoxOS_';
```
Comment (crh0716) [inline apps/settings/js/panels/hotspot/hotspot_settings.js]: nit: let's use `ssid`

File: apps/settings/elements/hotspot.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h2 data-l10n-id="internetSharing-wifi"></h2>
           <a class="info-item">
             <span data-l10n-id="wifi-name" class="info-item-title"></span>
             <span class="info-item-space"></span>
-            <span class="hotspotSSID info-item-content"></span>
+            <span class="hotspotSSID info-item-content" data-name="tethering.wifi.ssid"></span>
```
Comment (crh0716) [inline apps/settings/elements/hotspot.html]: This is not required as we already have a handler updating the span when SSID changes.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9722f5b034369370fc29c7b00f4f47f47d96c8d8

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f52cca0096d5f3c0bd0cadb1b9d5520a98622712

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#MfS2hEbMQbmBGy0BcHK70w

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=72c38751ba6ad10cb2bf27a5b7ed3ed685b51c64

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fa2ee7af6655dc51712a49e05b0620ccb729cfe1

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#Ovn9y9dGS4KwZODjo8b9-A

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request closed without merging**